### DISCUSSION TITLE
Support social links for Zulip

### DISCUSSION THREAD

**Discussion by kylewlacy** [withastro/starlight#1952] [Feature Requests]
### What version of `starlight` are you using?

0.22.3

### What is your idea?

Add support for setting a social link for [Zulip](https://zulip.com/) in the Starlight config, along with an appropriate icon

### Why is this feature necessary?

I just set up a Zulip for my project, but can't show it in the navigation header in Starlight

I saw Discourse was added recently (#1909) so I figured it wouldn't be unreasonable for Zulip to be supported as well

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by lorenzolewis** [comment]
Hi @kylewlacy ! If you were interested in submitting a PR for this then this would be a good reference to follow so you know which files to change: https://github.com/withastro/starlight/pull/1909/files 

(edit: Which I now clearly see that you already linked to ;) )

**Comment by kylewlacy** [comment]
Just opened a PR for this! Here it is: https://github.com/withastro/starlight/pull/1978

**Comment by delucis** [comment]
Thanks again for adding this @kylewlacy and to @lorenzolewis for helping out! :rocket:

Available as of Starlight v0.24.1
